packages/core/src/Stores/ui-store.js

```diff
@@ -33,7 +33,7 @@ export default class UIStore extends BaseStore {
     @observable settings_extension = undefined;
     @observable notification_messages_ui = undefined;
 
-    @observable is_dark_mode_on = false;
+    @observable is_dark_mode_on = window.matchMedia('(prefers-color-scheme: dark)').matches && isMobile();
```

Comment:
Shouldn't we prioritize `localStorage` values over?

Reply:
maybe we should do this for desktop as well. can you check with UX team about this?

Reply:
@salar-fs localStorage values are already prioritised, this condition is needed for default case when no localStorage settings found.

@balakrishna-binary I don't think we should do it for desktop because from my observation a lot of people use dark mode on desktop by default, sometimes without knowing it, so we better show them our normal theme initially.